classes/class-wc-rest-connect-packages-controller.php

```diff
@@ -8,51 +8,12 @@
 	return;
 }
 
-class WC_REST_Connect_Packages_Controller extends WP_REST_Controller {
+class WC_REST_Connect_Packages_Controller extends WC_REST_Connect_Base_Controller {
 
-	/**
-	 * Endpoint namespace.
-	 *
-	 * @var string
-	 */
-	protected $namespace = 'wc/v1';
-
-	/**
-	 * Route base.
-	 *
-	 * @var string
-	 */
+	protected $method = 'POST';
 	protected $rest_base = 'connect/packages';
 
-	/**
-	 * @var WC_Connect_Service_Settings_Store
-	 */
-	protected $service_settings_store;
-
-	/**
-	 * @var WC_Connect_Logger
-	 */
-	protected $logger;
-
-	public function __construct( WC_Connect_Service_Settings_Store $service_settings_store, WC_Connect_Logger $logger ) {
-		$this->service_settings_store = $service_settings_store;
-		$this->logger = $logger;
-	}
-
-	/**
-	 * Register the routes for settings.
-	 */
-	public function register_routes() {
-		register_rest_route( $this->namespace, '/' . $this->rest_base, array(
-			array(
-				'methods'             => WP_REST_Server::EDITABLE,
-				'callback'            => array( $this, 'update_items' ),
-				'permission_callback' => array( $this, 'update_items_permissions_check' ),
-			),
-		) );
-	}
-
-	public function update_items( $request ) {
+	public function run( $request ) {
 		$packages = $request->get_json_params();
 
 		$this->service_settings_store->update_packages( $packages[ 'custom' ] );
```

Comment:
I definitely missed this in my review - `$this->service_settings_store` is no longer with us.